fix_worktree_setup.sh

```
@@ -0,0 +1,121 @@
+#!/bin/bash
+# Fix Worktree Setup for Claude Code Parallel Development
+
+echo "🔧 Fixing Worktree Setup for Claude Code"
+echo "========================================"
+
+# Step 1: Free up main branch
+echo "📂 Step 1: Freeing up main branch..."
+if [ -d "worktrees/ci-fixes" ]; then
+    cd worktrees/ci-fixes
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)